server/datastore/mysql/software.go

```diff
@@ -696,6 +698,7 @@ func selectSoftwareSQL(opts fleet.SoftwareListOptions) (string, []interface{}, e
 				goqu.MAX("c.epss_probability").As("epss_probability"),     // for ordering
 				goqu.MAX("c.cisa_known_exploit").As("cisa_known_exploit"), // for ordering
 				goqu.MAX("c.published").As("cve_published"),               // for ordering
+				goqu.MAX("c.description").As("description"),               // for ordering
```

Comment:
MAX for description?

Reply:
I'm not a fan of the pattern, but this set of fields need to be some kind of aggregate type.  Maybe a future refactor?

Reply:
Does it not work if we don't put `MAX` for `description`?

Reply:
it complains that `description` needs to be an aggregate function

Reply:
Just worried about performance, but maybe not an issue given that a software piece won't have thousands of CVEs assigned, right?

Reply:
I think so, and if we get to that point we should think about paginating api responses

Reply:
We do paginate software. But yes, this is probably a-ok.